Microsoft.Azure.CosmosRepository/src/DefaultRepository.cs

```diff
@@ -9,6 +9,7 @@
 using System.Threading.Tasks;
 using Microsoft.Azure.Cosmos;
 using Microsoft.Azure.Cosmos.Linq;
+using Microsoft.Azure.CosmosRepository.Dtos;

```

Comment:
```suggestion
using Microsoft.Azure.CosmosRepository.Pagination;
```

I prefer the namespace of Pagination, instead of Dtos. Let's change this for scoping all of the pagination related features - as Dtos stands for "data transfer object" and doesn't apply directly to pagination.